styles.css

```diff
@@ -648,16 +648,129 @@ li.language {
   color: #fff;
 }
 
+.shape {
+  position: absolute;
+  bottom: 8px;
+  left: 45%;
+  width: 134px;
+  height: 15px;
+  border-radius: 10px;
+  background-color: #0d151e;
+}
+
 .logo-nav .nav-bar li:hover {
   background-color: #396df2;
   border-radius: 8px;
 }
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 You tried to add a `hover effect` to the mobile menu elements, this is a great idea 💯, but this `hover effect` is only fully visible in the `desktop version`, and not fully visible in the `mobile menu` elements. Please `fix` or `remove` this hover effect. Good luck 😃 

![image](https://user-images.githubusercontent.com/63915024/196483887-231d2d98-af4c-44bf-a448-6ce9367baca8.png)